src/main/java/com/github/sdonkov/PrefixTree.java

```diff
@@ -100,4 +106,8 @@ private TraverseResult traverse(CharSequence word) {
         }
         return traverseResult;
     }
+
+    private boolean checkIfNull(CharSequence word) {
+        return word == null;
+    }
```

Comment:
This looks weird. `word == null` is very common in Java code and is very readable. I can't see how wrapping it with a method improves anything.